.gitignore

```diff
@@ -14,3 +14,6 @@
 /.tox/
 /build-servers.yml
 /integration.html
+# NeatNerdPrime project files
+.project
+.pydevproject
```

Comment:
Thanks for pointing that out! Learned something new today, i added my global gitignore. It should be okay